dirty.html

```diff
@@ -1,205 +1,218 @@
-                </div>
-              </div>
-            </div>
-            <div>
-              <div>
-                <div>
-                  <div>Cancan</div>
-                  <div>Carrierwave</div>
-                  <div>Bcrypt</div>
-                  <div>Devise</div>
-                  <div>Sidekiq</div>
-                  <div>Textacular</div>
-                  <div>Factory Girl</div>
-                  <div>High Voltage</div>
-                  <div>Rspec</div>
-                  <div>Nokogiri</div>
-                  <div>Acts as Draggable</div>
-                  <div>Acts as Taggable</div>
-                  <div>Will Paginate</div>
-                  <div>Nested Forms</div>
-                  <div>Geocoder</div>
-                  <div>Jekyll</div>
-                  <div>Better Errors</div>
-                </div>
-              </div>
-            </div>
-            <div>
-              <div>
-                <div>
-                  <div>Javascript</div>
-                  <div>jQuery</div>
-                  <div>Coffeescript</div>
-                  <div>JSON</div>
-                  <div>AJAX</div>
-                  <div>Ember</div>
-                  <div>Jasmine</div>
-                  <div>Underscore.js</div>                    
-                  <div>Templates</div>
-                  <div>Responsive design</div>
-                  <div>HTML</div>
-                  <div>HAML</div>
-                  <div>Handlebars</div>
-                  <div>ERB</div>                    
-                  <div>CSS</div>
-                  <div>SCSS, SASS</div>
-                  <div>Bootstrap</div>
-                </div>
-              </div>
-            </div>
+          <!-- #concepts-column-2 -->
+          <div class="concepts-columns" id="concepts-column-2">
+            <ul>
+              <li>HTTP Requests</li>
+              <li>URL, URI</li>
+              <li>Networking</li>
+              <li>SSH</li>
+              <li>Sessions</li>
+              <li>Authentication</li>
+              <li>Authorization</li>
+              <li>Web Scraping</li>
+              <li>Cookies</li>
+              <li>HTTP APIs</li>
+              <li>RESTful APIs</li>
+              <li>HTTParty</li>
+              <li>Uploads</li>
+              <li>File I/O</li>
+              <li>Amazon S3 AWS</li>
+              <li>Sublime</li>
+              <li>Environment variables</li>
+            </ul>
           </div>
-        </div>
-        <div>
-          <div>
-            <div>The Instructors</div>
-            <a name="instructors"></a>
-            <div>They taught us everything we know</div> 
-            <div>Well, let's be honest. They taught us a lot more than we know, but some of it didn't commit to memory. But we are hackers now, thanks to these four.</div>
+          <!-- end of #concepts-column-2 -->
+
+          <!-- #concepts-column-3 -->
+          <div class="concepts-columns" id="concepts-column-3">
```

Comment:
you should only assign `id` when you want to target an element for styling or linking to it. Think of being minimalistic, meaning just write enough code to make it work.

> **I would have written a shorter letter, but I did not have the time.**
> _Provincial Letters: Letter XVI, 4 December, 1656_  | http://en.wikiquote.org/wiki/Blaise_Pascal